lang/javascript.py

```diff
@@ -1,120 +1,101 @@
 from talon.voice import Context, Key, press
 import talon.clip as clip
-from ..utils import (
-    text,
-    parse_words,
-    parse_words_as_integer,
-    insert,
-    word,
-    join_words,
-    is_filetype,
-)
+from ..utils import text, parse_words, parse_words_as_integer, insert, word, join_words
 
-JS_EXTENSIONS = (".js", ".jsx")
-
-context = Context("javascript", func=is_filetype(JS_EXTENSIONS))
+def verifyExtension(app, win):
+    return win.doc.endswith(".js") or win.doc.endswith(".jsx")
 
+context = Context("javascript")
```

Comment:
with commands like `let` and `call`, i don't think these commands can be in the global context. Perhaps constraints of file types and Chrome like the other language scripts. More long-term we might want some configuration options about which languages stay in global scope.

Reply:
Fixed this by utilizing the verifyExtension function with context.